client/src/pages/Reviews/Review.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ReviewStyled, ReviewText } from "./ReviewStyled";
+import  ReviewsStyled from "./ReviewsStyled"
+import { reviewSeed } from "./reviewSeed";
+
+const payloadList = reviewSeed.map(({ user, rating, keyword, value, reviewText }) => {
```

Comment:
I would just use an object here as the prop and call it `review`.

Reply:
Ok, will do.